tailwind.config.cjs

```diff
@@ -3,16 +3,23 @@ module.exports = {
   content: [
     "./src/pages/**/*.{js,ts,jsx,tsx,mdx}",
     "./src/components/**/*.{js,ts,jsx,tsx,mdx}",
+    "./src/styles/**/*.{js,ts,jsx,tsx,mdx}",
+    "./src/atoms/**/*.{js,ts,jsx,tsx,mdx}",
     "./src/app/**/*.{js,ts,jsx,tsx,mdx}"
   ],
   theme: {
     extend: {
+      fontSize: {
+        sm: '16px'
```

Comment:
We already have a '16px' text size and that's `text-base`. What is the reasoning or motivation behind redefining `text-sm` to be '16px' (1 rem) instead of its standard '14px' (0.875 rem)?